pyinteraph/path_analysis.py

```diff
@@ -164,6 +169,12 @@ def sort_paths(graph, paths, sort_by):
     sorted_paths = sorted(paths, key = key, reverse = reverse)
     return sorted_paths
 
+def write_table(fname, table):
+    """Save sorted table as txt file. """
+    with open(fname, "w") as f:
+        for p, s, t, l, sum_w, avg_w in table:
```

Comment:
Use more meaningful variable names here instead of single lowercase letters (more difficult to debug later on).